tests/nnstreamer_query/runTest.sh

```diff
@@ -41,46 +41,75 @@ if [[ ! $check_query ]]; then
     exit
 fi
 
+function waitformarker {
+  for i in $(seq 1 ${TIMEOUT_SEC})
+  do
+    if [ -f 'marker.log' ]; then
+      markersize=$(stat -c%s marker.log)
+      if [ $markersize -ge 48 ]; then
+        testResult 1 $1 "$2" 0 0
+        return 0
+      fi
+    fi
+    sleep 1
+  done
+  testResult 0 $1 "$2" 0 0
+  exit
+}
+WAIT4MARKER=" videotestsrc num-buffers=1 ! video/x-raw,format=RGB,height=4,width=4 ! filesink location=marker.log "
+PORT1=$(expr $RANDOM % 1000 \* 2 + 5000)
+PORT2=$(expr $PORT1 + 1)
```

Comment:
How about using this to avoid unavailable port number?
```suggestion
PORT1=`python3 ../nnstreamer_grpc/get_available_port.py`
PORT2=`python3 ../nnstreamer_grpc/get_available_port.py`
```

Reply:
Applied.